addons/survey/models/survey_question.py

```diff
@@ -275,6 +280,45 @@ def _compute_validation_required(self):
             if not question.validation_required or question.question_type not in ['char_box', 'numerical_box', 'date', 'datetime']:
                 question.validation_required = False
 
+    @api.depends('is_conditional', 'survey_id', 'survey_id.question_ids', 'triggering_question_id')
+    def _compute_allowed_triggering_question_ids(self):
+        """ This method is required to fetch the possible triggering questions when
+        the question is being created. """
+        conditional_questions = self.filtered(lambda q: q.is_conditional)
+        non_conditional_questions = self - conditional_questions
+        non_conditional_questions.allowed_triggering_question_ids = False
+        non_conditional_questions.is_placed_before_trigger = False
+        if not conditional_questions:
+            return
+
+        possible_trigger_questions = self.search([
+            ('is_page', '=', False),
+            ('question_type', 'in', ['simple_choice', 'multiple_choice']),
+            ('suggested_answer_ids', '!=', False),
+            ('survey_id', 'in', self.survey_id.ids)
+        ])
+        # Make sure to use sequence stored in db for existing questions passed as newIds.
+        # This works well as we are updating records each time a question form is opened/closed.
+        self.env.cr.execute(
+            "SELECT id, sequence FROM survey_question WHERE id =ANY(%s)",
+            [conditional_questions.ids]
+        )
+        conditional_questions_sequences = dict(self.env.cr.fetchall())  # id: sequence mapping
+
+        for question in conditional_questions:
+            question_id = question._origin.id
+            # question.sequence is used for newly created questions where the cache's value is
+            # always higher than all existing questions.
+            question_sequence = conditional_questions_sequences[question_id] or question.sequence
```

Comment:
Mmmh, this seems to not always work.

Latest runbot, new survey, created a first question, choice, with 3 answers.
Created second question, checked the "Conditional Display" box, and boom:

![image](https://user-images.githubusercontent.com/41282536/210972285-03926772-15e7-40b8-9921-820d93121ada.png)

Maybe a tour test would be ideal to make sure everything works, as current code depends on both JS code and backend computed fields?